Index: lib/pages/home/produits.dart
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/pages/home/produits.dart b/lib/pages/home/produits.dart
--- a/lib/pages/home/produits.dart	
+++ b/lib/pages/home/produits.dart	(date 1718664467550)
@@ -15,35 +15,33 @@
 
   @override
   void initState() {
-      super.initState();
-      http.get(Uri.parse("http://192.168.228.44:9000/produits"))
-      .then((resp){
-        // print(resp.body);
-        setState(() {
-          produits = json.decode(resp.body);
-        });
-      })
-      .catchError( (erreur) {
-        print('********** Erreur *********');
-        print(erreur);
-      });
+    super.initState();
+    http.get(Uri.parse("http://192.168.228.44:9000/produits")).then((resp) {
+      // print(resp.body);
+      setState(() {
+        produits = json.decode(resp.body);
+      });
+    }).catchError((erreur) {
+      print('********** Erreur *********');
+      print(erreur);
+    });
   }
+
   @override
   Widget build(BuildContext context) {
-    return   Scaffold(
-      appBar: AppBar( title:  const Text("Produits"),),
-      body: ListView.builder(
-        itemCount: produits.length,
-        itemBuilder: (context, index){
-          // print(index);
-          return
-            ListTile(
-            title: Text("${produits[index]["nom"]}"),
-          )
-          ;
-        },
-      )
-    );
+    return Scaffold(
+        appBar: AppBar(
+          title: const Text("Produits"),
+        ),
+        body: ListView.builder(
+          itemCount: produits.length,
+          itemBuilder: (context, index) {
+            // print(index);
+            return ListTile(
+              title: Text("${produits[index]["nom"]}"),
+            );
+          },
+        ));
   }
 }
 
@@ -67,4 +65,4 @@
 //       ),
 //     );
 //    } // build
-// }
\ No newline at end of file
+// }
